libraries/botbuilder-schema/botbuilder/schema/teams/teams_channel_data.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+
+from botbuilder.schema.teams import ChannelInfo, TeamInfo, NotificationInfo, TenantInfo
+
+
+class TeamsChannelData:
+    def __init__(
+        self,
+        channel: ChannelInfo = None,
+        eventType="",
```

Comment:
Why not "event_type" instead of "eventType"?

Reply:
Ahh the comment on this one must have been lost. I'll re-add it.

It's this way because I'm using the flat map method (**) to create objects based on the JSON that's coming in on the activity. The name of the field in the JSON is eventType so that's what it has to be here. IIRC the property on the class is correctly cased.